osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -513,9 +519,11 @@ private void updateButtonsState()
             int loadedRepliesCount = loadedReplies.Count;
             bool hasUnloadedReplies = loadedRepliesCount != Comment.RepliesCount;
 
+            showRepliesButton.FadeTo(loadedRepliesCount != 0 ? 1 : 0);
             loadRepliesButton.FadeTo(hasUnloadedReplies && loadedRepliesCount == 0 ? 1 : 0);
+            repliesButtonContainer.FadeTo(showRepliesButton.IsPresent || loadRepliesButton.IsPresent ? 1 : 0);
```

Comment:
It's in a fill flow and spacing is applied even on `0x0` drawables, yes.

![osu!_fKdUS5x56S](https://user-images.githubusercontent.com/35318437/216576145-eae40c68-325f-4cf6-90e4-5cd97b734e9f.png)

Reply:
Can agree that using `IsPresent` for any logic outside of framework is not great. Also this is doubly not-great because what you _really_ want here is "hide the parent container if its all children are also hidden" logic, but that is being achieved by manually checking both children.

If I were to suggest a quick fix, it would be

```suggestion
            repliesButtonContainer.FadeTo(repliesButtonContainer.Any(child => child.Alpha > 0) ? 1 : 0);
```